elephant/causality/granger.py

```diff
@@ -596,3 +776,163 @@ def conditional_granger(signals, max_order, information_criterion='aic'):
                                                  est_sig_figures)
 
     return conditional_causality_xy_z_round
+
+
+def pairwise_spectral_granger(signal_i, signal_j, fs=1, nw=4, num_tapers=None,
+                              peak_resolution=None, n_segments=1,
+                              len_segment=None, frequency_resolution=None,
+                              overlap=0.5, num_iterations=300):
+    r"""Determine spectral Granger Causality of two signals.
+
+    The spectral Granger Causality is obtained through the following steps:
+
+    1. Determine the cross spectrum of the two signals by applying
+       :func:`segmented_multitaper_cross_spectrum` to the joint signal. See the
+       documentation of this function for the parameter hierarchy of the
+       used for the estimation of the cross spectrum.
+
+    2. Calculate the spectral factorization of the cross spectrum decomposing
+       the very same thing into the covariance matrix and the transfer
+       function.
+
+    3. Calculate the directional and instantaneous spectral Granger Causality
+       from the power spectra, the transfer function and the covariance matrix.
+
+    Parameters
+    ----------
+    signal_i : neo.AnalogSignal or pq.Quantity or np.ndarray
+        First time series data of the pair between which spectral Granger
+        Causality is computed.
+    signal_j : neo.AnalogSignal or pq.Quantity or np.ndarray
+        Second time series data of the pair between which spectral Granger
+        Causality is computed.
+        The shapes and the sampling frequencies of `signal_i` and `signal_j`
+        must be identical. When `signal_i` and `signal_j` are not
+        `neo.AnalogSignal`, sampling frequency should be specified through the
+        keyword argument `fs`. Otherwise, the default value is used
+        (`fs` = 1.0).
+    fs : float, optional
+        Specifies the sampling frequency of the input time series
+        Default: 1.0
+    nw : float, optional
+        Time bandwidth product
+        Default: 4.0
+    num_tapers : int, optional
+        Number of tapers used in 1. to obtain estimate of PSD. By default,
+        [2*nw] - 1 is chosen.
+        Default: None
+    peak_resolution : pq.Quantity float, optional
+        Quantity in Hz determining the number of tapers used for analysis.
```

Comment:
This  is spelled out in detail in the documentation `segmented_multitaper_cross_spectrum`, to which we link in the first bulletpoint in the documentation of the function.
The reason is to avoid a duplication.
Do you think it is necessary here?

Reply:
I see this was already cross referenced earlier in the docstring